src/main/java/org/checkerframework/specimin/PrunerVisitor.java

```diff
@@ -156,13 +156,15 @@ public Visitable visit(ConstructorDeclaration constructorDecl, Void p) {
     ResolvedConstructorDeclaration resolved = constructorDecl.resolve();
     if (methodsToLeaveUnchanged.contains(resolved.getQualifiedSignature())) {
       return super.visit(constructorDecl, p);
-    } else if (membersToEmpty.contains(resolved.getQualifiedSignature())) {
```

Comment:
Can you clarify why it's preferable to isolate these conditional statements rather than employing an if-elseif-else clause? I think using such a clause would enhance the clarity of logical connections within the code elements.